app/editor/extensions/MentionMenu.tsx

```diff
@@ -8,7 +8,7 @@ export default class MentionMenuExtension extends Suggestion {
   get defaultOptions() {
     return {
       // ported from https://github.com/tc39/proposal-regexp-unicode-property-escapes#unicode-aware-version-of-w
-      openRegex: /(?:^|\s|\()@([\p{L}\p{M}\d]+)?$/u,
+      openRegex: /(?:^|\s|\()@([\p{L}\p{M}\d@\.]+)?$/u,
```

Comment:
Quite a loose regex for handling emails but I guess it should be fine for the user case? 🤔